py/janitor/site/simple.py

```diff
@@ -705,12 +705,18 @@ async def main_async(argv=None):
     public_runner = web.AppRunner(public_app)
     await private_runner.setup()
     await public_runner.setup()
+
     site = web.TCPSite(private_runner, args.host, port=args.port)
     await site.start()
-    logging.info("Listening on %s:%s", args.host, args.port)
+    logging.info("Private aiohttp listening on %s:%s", args.host, args.port)
+
     site = web.TCPSite(public_runner, args.host, port=args.public_port)
     await site.start()
-    logging.info("Listening on %s:%s", args.host, args.public_port)
+    logging.info("Public aiohttp listening on %s:%s", args.host, args.public_port)
+
+    if args.debugtoolbar:
+        logging.info("Debug toolbar enabled for %s: http://%s:%s/_debugtoolbar", args.debugtoolbar, args.host, args.public_port)
```

Comment:
Not sure I follow why this is explicitly printed? The debugtoolbar is integrated into the site by default, there is no need to go to this particular URL.

Reply:
The URL should be irrelevant here, the toolbar integrates with the pages served.

Reply:
If you ran --debugtoolbar for the first time, would you know where/what todo afterwards?

That was my reasoning for adding it.